internal/shipwright/build/build.go

```diff
@@ -68,7 +68,7 @@ func (sb *ShipwrightBuild) CreateOrUpdate(ctx context.Context, owner client.Obje
 
 	return ctrl.CreateOrUpdate(ctx, sb.Client, object, func() error {
 		object.Spec = shipwrightv1alpha1.ShipwrightBuildSpec{
-			TargetNamespace: common.OpenShiftBuildNamespaceName,
+			TargetNamespace: owner.GetNamespace(),
```

Comment:
IIRC the `ShipwrightBuild` object is cluster-scoped, so `GetNamespace()` will always return an empty string.

Reply:
Yes, `ShipwrightBuild` namespace will be blank. 
You can use the downward api to populate an environment variable or more convenient you can get it from `/var/run/secrets/kubernetes.io/serviceaccount/namespace` inside the container.